src/main/groovy/io/seqera/wave/service/builder/BuildRequest.groovy

```diff
@@ -149,7 +149,7 @@ class BuildRequest {
         this.buildContext = buildContext
         this.condaFile = condaFile
         this.spackFile = spackFile
-        this.targetImage = makeTarget(format, repo, containerId, condaFile, spackFile)
+        this.targetImage = makeTarget(format, repo, containerId, condaFile, spackFile, imageName)
```

Comment:
I am missing something here.  Where does `imageName` come from?

I would have expected it to come as a new last argument of this `BuildRequest` initialiser method (line 145 above), considering  call to this method has been modified accordingly in `ContainerTokenController` lines 243-259.

Reply:
yes sorry, it was deleted while resolving the conflicts added back